src/pages/_error.jsx

```diff
@@ -5,14 +5,13 @@
 
 import BaseLayout from 'layouts/base-layout'
 import MobileMenuLevelsGeneric from 'components/mobile-menu-levels-generic'
-import proxiedLayouts from 'layouts/_proxied-dot-io/dict'
 import ErrorViewSwitcher from 'views/error-view-switcher'
 // product data, needed to render top navigation
 import { productConfig } from 'lib/cms'
 import { isProductSlug } from 'lib/products'
 import { HOSTNAME_MAP } from 'constants/hostname-map'
 
-function Error({ statusCode, proxiedProductSlug }) {
+function Error({ statusCode }) {
```

Comment:
Can we also get rid of the giant comment here?

Reply:
We sure can! https://github.com/hashicorp/dev-portal/pull/2338/commits/e75873d04f11a672c2d8a1184238cffb0efdb434